debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/AbstractMySqlFieldReader.java

```diff
@@ -49,7 +49,15 @@ public Object readField(ResultSet rs, int columnIndex, Column column, Table tabl
         // by reading them with a calendar based on the default time zone, we make sure that the value
         // is constructed correctly using the database's (or connection's) time zone
         if (column.jdbcType() == Types.TIMESTAMP) {
-            return readTimestampField(rs, columnIndex, column, table);
+            try {
+                return readTimestampField(rs, columnIndex, column, table);
+            }
+            catch (RuntimeException e) {
+                logger.warn(String.format("Failed to read timestamp value: '%s'. Trying default ResultSet implementation.", e.getMessage()));
```

Comment:
@vjuranek Could you please use standard `{}` formatting placeholder in log message instead of `String.format`?

Reply:
Sure, fixed, @jpechane